src/routes/github/webhook/webhook-receiver-post.ts

```diff
@@ -74,7 +74,7 @@ export const WebhookReceiverPost = async (request: Request, response: Response):
 };
 
 const webhookRouter = async (context: WebhookContext) => {
-	const VALID_PULL_REQUEST_ACTIONS = ["opened", "reopened", "closed", "edited"];
+	const VALID_PULL_REQUEST_ACTIONS = ["opened", "reopened", "closed", "edited", "converted_to_draft", "ready_for_review"];
```

Comment:
Did you test these actions? It looks like it's `convert_to_draft` not converted.

![Screen Shot 2023-01-19 at 11 45 47 am](https://user-images.githubusercontent.com/37155488/213329608-b3cdf36c-15d5-4353-b2e5-8856a664e38f.png)